heron/api/src/java/com/twitter/heron/streamlet/SerializableGenerator.java

```diff
@@ -0,0 +1,29 @@
+//  Copyright 2017 Twitter. All rights reserved.
```

Comment:
Whats this new file?

Reply:
Both of these files (along with two others) seem to be results of errant merges.